docs/developer/developer-manual.html

```diff
@@ -161,6 +161,24 @@ <h2 id="Build_tasks">Build tasks</h2>
 main directory or any subproject runs the <code>allTests</code> task only in the <code>framework</code> project.
 </p>
 
+<p>
+An iterative build-test-debug cycle where you assemble the Checker Framework
+and execute its tests can sometimes lead to a state where Gradle is not able to
+compile and build the system for no obvious reason.
+This is especially common when you kill tests before they fully finish, or kill
+compilation before it is fully complete, leading to a bad state or an
+inconsistent build cache.
+</p>
+
+<p>
+Below are some steps you might try to resolve this issue (in order):
+</p>
+<ul>
+  <li><code>./gradlew --no-build-cache assemble</code>: attempts to rebuild the Checker Framework without the build cache.
+  <li>Kill any Gradle daemons; run <code>./gradlew --status</code> to get a list of pids for Gradle.
```

Comment:
I usually just do `killall java`, but that will kill other java processes as well.  Conveniently, it _won't_ kill IntelliJ.

Reply:
Killing only the Gradle daemons is safer.  It'd be nice to figure out a one-liner to kill all of them like with xargs or something.